_episodes/02-importing-data.md

```diff
@@ -31,11 +31,12 @@ keypoints:
 >1. Once OpenRefine is launched in your browser, click `Create Project` from the left hand menu and select `Get data from This Computer`
 >2. Click `Choose Files` (or 'Browse', depending on your setup) and locate the file which you have downloaded called `doaj-article-sample.csv`
 >3. Click `Next >>` - the next screen (see below) gives you options to ensure the data is imported into OpenRefine correctly. The options vary depending on the type of data you are importing.
->4. Click in the `Character encoding` box and set it to `UTF-8`
+>4. Click in the `Character encoding` box and set it to `UTF-8` as it will ensure the special characters in the Author column are displaying correctly
```

Comment:
The behaviour with this specific project varies across OpenRefine versions, and also this will fix characters in all fields (even if it is the Author column where the most obvious problems are, there could be issues in other columns that we just aren't seeing the preview).

So suggest this is made a slightly more general comment about "this ensures that OpenRefine correctly interprets the imported data as UTF-8 encoded. If you don't select this you may find that some special characters (e.g. "smart quotes") are not displayed correctly" (please feel free to re-word, just giving an example to be clear)